dsgrid/config/dataset_config.py

```diff
@@ -444,17 +494,18 @@ def check_time_zone(self) -> "DatasetConfigModel":
 def make_unvalidated_dataset_config(
     dataset_id,
     metric_type: str,
-    table_format: dict[str, str] | None = None,
-    data_classification=DataClassificationType.MODERATE.value,
-    dataset_type=InputDatasetType.MODELED,
+    table_format: TableFormatModelBase = UnpivotedTableFormatModel(),
```

Comment:
This violates the rule of assigning a mutable value to a default parameter. This is ok if we don't change it. But maybe someone will change it in the future. This is why I usually default to None in this case.

Reply:
I'll defer to you.